tests/conftest.py

```diff
@@ -1,9 +1,26 @@
+import os
+
 import numpy as np
 import pandas as pd
 
 import pytest
 
 
+def maybe_convert_to_polars(df):
```

Comment:
updating several fixtures to return `polars.DataFrame` instead of `pandas.DataFrame`